src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeEventRepositoryTest.java

```diff
@@ -42,23 +43,22 @@ void should_save_and_read_events() {
         var event2 = new NewEnvelopeEvent(envelopeId, EventType.DELETED, "note 2");
 
         // when
-        eventRepo.insert(event1);
-        eventRepo.insert(event2);
+        long eventId1 = eventRepo.insert(event1);
+        long eventId2 = eventRepo.insert(event2);
 
         var eventsInDb = eventRepo.findForEnvelope(envelopeId);
 
         // then
-        assertThat(eventsInDb).hasSize(2);
+        assertThat(eventsInDb)
```

Comment:
Changed the assertion as the order of the records is different from the expectation